plugins/scaffolder/src/next/Router/Router.tsx

```diff
@@ -84,6 +86,8 @@ export const Router = (props: PropsWithChildren<NextRouterProps>) => {
       TemplateCardComponent,
       TemplateOutputsComponent,
       TaskPageComponent = OngoingTask,
+      TemplatePageHeaderComponent,
```

Comment:
I'm feeling that we should just be replacing the page here. These `props` feel quite fine grained control of something that isn't really a common use case to replace in any plugin.

We've also got prior art for replacing the entire page with a `prop` in this interface, so let's do that instead.